[MPDX-8322] Update translations

canac opened the pull request:
## Description

* Update translation files
* Sort translation files so that future diffs will be easier to read

[MPDX-8322](https://jira.cru.org/browse/MPDX-8322)

## Checklist:

- [ ] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [ ] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [ ] I have requested a review from another person on the project

dr-bizz reviewed (commented):
It looks like we cost just under 300 translations. was that expected?

We lost words like `Reminders`.

canac commented:
> It looks like we cost just under 300 translations. was that expected?

@dr-bizz Where are you getting the number 300 from? `git diff main --stat -- public/locales/en/translation.json` says we removed 15 labels and added 39. All the changes to `public/locales/en/translation.json` look correct.

dr-bizz commented:
> @dr-bizz Where are you getting the number 300 from? git diff main --stat -- public/locales/en/translation.json says we removed 15 labels and added 39. All the changes to public/locales/en/translation.json look correct.

I'm getting the number 300 from the number of lines that have been removed.

dr-bizz reviewed (approved):
I will approve this as I don't want to block you.
